### ISSUE TITLE
Missing Bootstrap

### ISSUE THREAD

**Issue by PipBoy2000** [jerisalan/transit-app#1]
FYI, get this error when running:

./src/index.js
Module not found: Can't resolve 'bootstrap/dist/css/bootstrap.css' in 'C:\Users\Dennis\SF-Transit-master\src'

The entire dist directory is missing in the project, it seems.

**Comment by jerisalan** [comment]
Did you do a npm install before trying to run the node.js script?

**Comment by PipBoy2000** [comment]
Yes.  I don't see bootstrap in the dependencies.  And again, the dist directory isn't in the project.  Are you saying it should be built by the install?

**Comment by jerisalan** [comment]
I am not sure if that is the correct way to start a react project. `npm start` works perfectly all right.

**Comment by PipBoy2000** [comment]
No, it doesn't.  

You might wish to consider unpublishing this project since it is having missing dependencies/eslint errors.  Just a piece of advice.

**Comment by jerisalan** [comment]
I just did a fresh clone of the project and did a npm start after npm install. There is no missing dependency. Like I said it was a learner project and I really am not sparing any time for fixing this now.
